packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/namespaces/add/AddNamespace.modal.tsx

```diff
@@ -0,0 +1,22 @@
+import { useServiceData } from '../../Service.context';
+import AddEditNamespace from '../_components/AddEditNamespace.component';
+import { Skeleton } from '@/components/ui/skeleton';
+import { useGetNamespaces } from '@/hooks/api/database/namespace/useGetNamespaces.hook';
+
+const AddNamespaceModal = () => {
+  const { projectId, service } = useServiceData();
+  const namespacesQuery = useGetNamespaces(
+    projectId,
+    service.engine,
+    service.id,
+    {
+      enabled: !!service.id,
+    },
+  );
+  const namespaces = namespacesQuery.data;
+
+  if (!namespacesQuery.data) return <Skeleton className="w-full h-4" />;
```

Comment:
is the error handled somewhere ?

Reply:
We have a error boundary component that automatically handle errors